jxl/src/image.rs

```diff
@@ -189,7 +189,7 @@ pub mod debug_tools {
 
     impl ToU8ForWriting for u16 {
         fn to_u8_for_writing(self) -> u8 {
-            (self as u32 * 255 / 65535) as u8
+            (self as u32 >> 8) as u8
```

Comment:
My impression is that what would likely make the most sense is something along the lines of: `((self as u32 * 255 + 32767) / 65535) as u8` (roughly equivalent to converting to a real between 0 and 255 and rounding it). How about it?